docs/source/torch_environment_variables.rst

```diff
@@ -0,0 +1,20 @@
+.. _torch_environment_variables:
+
+Torch Environment Variables
+===============================
+PyTorch uses environment variables to control various settings, such as which
+gloo device to use, the number of threads used for parallelism, and the number
+of OpenMP threads, and many others. As well, some libraries that PyTorch uses
+(e.g., MKL) also use environment variables to control their behavior. This
+page lists the environment variables that can be used to configure PyTorch.
+Note: There are many environment variables and this list is not exhaustive.
```

Comment:
Maybe a comment of "If you find anything on this doc out of date, please open an issue at <link to our issue creation page>!"